### PR TITLE
Material Changes

### PR DESCRIPTION
**Pull request by xcthulhu**
- Updating image names and locations
- Updates to staking documentation
- Updates for using docker for running nodes

### CODE REVIEW

**Review by uchu — CHANGES_REQUESTED** [review]
@xcthulhu Changes are amazing, however I have a few notes.

I've also decided to stay with Honkit, as Gatsby works fine, but we can achieve same look and feel with less efforts with Honkit. So I made a step back from Gatsby and will make the changes including to the theme switching, etc.

File: src/sydney-testnet/validator-guides/how-to-chill.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# How to Pause Staking
+
+The Golden Gate validator may opt to pause staking (or *chill*) whenever he or she wants to prevent participation in staking for the upcoming era but does not want to unstake funds (e.g. when there is maintenance planned).
+
+Validator can do that by calling the `chill` extrinsic in the [staking pallet](https://paritytech.github.io/substrate/master/pallet\_staking/pallet/enum.Call.html#variant.chill) using following the steps:
```
[the remark this answers is not in the document]
Comment (xcthulhu) [reply src/sydney-testnet/validator-guides/how-to-chill.md]: I got rid of this link

File: package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "scripts": {
+    "build": "honkit build",
+    "serve": "honkit serve"
+  },
+  "devDependencies": {
+    "honkit": "^4.0.8"
```
[the remark this answers is not in the document]
Comment (xcthulhu) [reply package.json]: Fixed

File: src/sydney-testnet/validator-guides/how-to-stake-ggxt-tokens.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# How to Stake GGXT Tokens
+
+To stake GGXT tokens in the Golden Gate Sydney testnet, please follow the steps:
+
+1. Acquire tokens. Please check out our [Sydney Validator Program](../sydney-testnet/sydney-validator-programme.md "Sydney Validator Program").
+2. Create two substrate accounts. You can do that using [PolkadotJS](../../developer-documentation/wallets/polkadotjs.md).
```
[the remark this answers is not in the document]
Comment (xcthulhu) [reply src/sydney-testnet/validator-guides/how-to-stake-ggxt-tokens.md]: Fixed

File: src/developer-documentation/how-to-guides/how-to-deploy-rust.md [hunk 4]
```diff
@@ -12,6 +12,6 @@ To deploy Rust contract on the Golden Gate network you can follow video tutorial
 **Deploy your contract on the Golden Gate**
 
 1. Navigate to explorer of the Golden Gate network you want to deploy your contract on ([https://testnet.brooklyn.ggxchain.io/](https://testnet.brooklyn.ggxchain.io/) for the Brooklyn testnet)
-2. Go to Developer-->Contracts page
+2. Go to Developer 🡒 Contracts page
 3. Press Upload & deploy code
 4. Choose deployment account (you will need PolkadotJS or other substrate account manager to use your account)
```
[the remark this answers is not in the document]
Comment (xcthulhu) [reply src/developer-documentation/how-to-guides/how-to-deploy-rust.md:17]: Fixed

File: src/sydney-testnet/validator-guides/how-to-setup-a-validator-node.md [hunk 5]
```diff
@@ -28,33 +43,4 @@ You can use the following optional flags:
 | `--chain <chain-specification>`   | Specifies the chain specification to use. You can set this option using a predefined chain specification name,such as `dev`, `local`, or `staging`or you can specify the path to a file that contains the chain specification, for example, the chain specification generated by using the build-spec subcommand.                                                                                                                                      |
```
[the remark this answers is not in the document]
Comment (xcthulhu) [reply src/sydney-testnet/validator-guides/how-to-setup-a-validator-node.md:42]: Fixed

File: src/sydney-testnet/sydney-bug-bounty-program.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 The Sydney bug bounty program is an initiative by the Golden Gate team to engage web3 developers and enthusiasts in securing the Golden Gate network for a reward in GGXT tokens.
 
-The scope of the program is the Golden Gate Sydney testnet.&#x20;
+The scope of the program is the Golden Gate Sydney testnet.
 
 **How to Apply**
```
[the remark this answers is not in the document]
Comment (xcthulhu) [reply src/sydney-testnet/sydney-bug-bounty-program.md:9]: Done

File: src/sydney-testnet/validator-guides/how-to-stake-ggxt-tokens.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+Make sure not to bond all your GGXT balance since you cannot pay transaction fees from your bonded balance.
+
+#### Session keys
+
+To get your node nominated, please follow the steps:
+
+1.  Get your node session key. Assuming you are running using docker, you can do that running the command:
+
+    ```bash
+    docker exec -it goofy_curie curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "author_rotateKeys", "params":[]}' http://localhost:9933
```
[the remark this answers is not in the document]
Comment (xcthulhu) [reply src/sydney-testnet/validator-guides/how-to-stake-ggxt-tokens.md]: done

**Review by uchu — APPROVED** [review]
LGTM

**Pull request merged**